internal/InfrastructureManagerConfig.go

```diff
@@ -0,0 +1,74 @@
+package internal
+
+import (
+	"encoding/json"
+	"io"
+)
+
+type InfrastructureManagerConfig struct {
```

Comment:
In general the structure is a bit inconsistent. We have a nesting struct `ConverterConfig` that contains settings used by the converter. On the other hand we have `DefaultSharedIASTenant` that is on the same level. In general there are two options for the config design:
- `ConverterConfig` contains structures that are a bag for specific settings (e.g. converter settings)
- There is no nested object, and everything is under `ConverterConfig`

Currently we are somewhere in between. Flattening the configuration is not an option so we could:
- leave it as it is
- come up with some nested object  that will contain `DefaultSharedIASTenant` (e.g. cluster)

Reply:
I went for the second option